model/change_point_detection.go

```diff
@@ -57,6 +59,34 @@ var (
 	perfAlgorithmOptionValueKey = bsonutil.MustHaveTag(AlgorithmOption{}, "Value")
 )
 
+type TriageInfo struct {
+	TriagedOn time.Time    `bson:"triaged_on" json:"triaged_on" yaml:"triaged_on"`
+	Status    TriageStatus `bson:"triage_status" json:"triage_status" yaml:"triage_status"`
+}
+
+var (
+	perfTriageInfoTriagedOnKey = bsonutil.MustHaveTag(TriageInfo{}, "TriagedOn")
+	perfTriageInfoStatusKey    = bsonutil.MustHaveTag(TriageInfo{}, "Status")
+)
+
+type TriageStatus string
+
+const (
+	TriageStatusUntriaged          TriageStatus = "untriaged"
```

Comment:
I think it makes sense to use the empty string "" as the untriaged value.

Reply:
Design doc dictates "untriaged", and I'd really prefer this explicit label vs an empty one.